packages/patternfly-4/react-core/src/components/Modal/ModalBoxHeader.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import accessibleStyles from '@patternfly/patternfly/utilities/Accessibility/accessibility.css';
+import { css } from '@patternfly/react-styles';
+
+import { Title, TitleLevel } from '../Title';
+
+export interface ModalBoxHeaderProps {
+  /** content rendered inside the Header */
+  children?:React.ReactNode;
+  /** additional classes added to the button */
+  className?: string;
+  /** Flag to hide the title */
+  hideTitle?: boolean;
+  /** the heading level to use */
+  headingLevel?: TitleLevel;
+}
+
+export const ModalBoxHeader: React.FunctionComponent<ModalBoxHeaderProps> = ({
+  children = null,
+  className = '',
+  hideTitle = false,
+  headingLevel = TitleLevel.h1,
```

Comment:
Nice addition, but let's leave the default as `TitleLevel.h3` like it was in the JS.

Reply:
There was an issue filed (#1736) asking for this to be an h1 by default